containers/scripts/build_strongswan.sh

```diff
@@ -274,6 +278,13 @@ case "x${BUILD_MODE}" in
 		build_strongswan_deb_for_client
 		;;
 
+	"xbuild-strongswan-client-apk")
```

Comment:
Also update 'clean_strongswan' to clean the .apk, the apk_build directory and any other file created during the build